content/en/apps/reference/app-settings/user-permissions.md

```diff
@@ -23,6 +23,7 @@ Each permission is defined as an array of user role identifiers that have the pe
 | `can_aggregate_targets` | Allows access to Target Aggregates page |
 | `can_bulk_delete_reports` | Allows users to select multiple reports and delete |
 | `can_configure` | Allows update of configuration parameters |
+| `can_upgrade` | Allows upgrade of the CHT instance |
```

Comment:
`upgrade` here could be mistaken for workflow upgrades, ie: configuration. How about we change it to...

```suggestion
| `can_upgrade` | Allows upgrades of the CHT Core Framework version via the API or admin interface |
```